File: Valimised/src/com/valimised/shared/Candidate.java
package com.valimised.shared;


public class Candidate {
	
	private int id;
	
	private String firstName;
	
	private String lastName;
	
	private String party;
	
	private String area;
	
	private String description;
	
	private String image;
	
	private int votes;
	
	public Candidate(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
	}

	public Candidate(String lastName, String firstName, int id, String area,
			String party) {
		this.firstName = firstName;
		this.lastName = lastName;
		this.id = id;
		this.area = area;
		this.party = party;
	}
	
	public Candidate(String lastName, String firstName, int id, String area,
			String party, String description, String image) {
		this.firstName = firstName;
		this.lastName = lastName;
		this.id = id;
		this.area = area;
		this.party = party;
		this.description = description;
		this.image = image;
	}
}

File: Valimised/war/javascript/candidate.js
function createCandidate(firstName, lastName, area, party, idCode) {
	$.ajax({
		url:'/application?firstName=' + firstName + '&lastName=' + 
			lastName + '&area=' + area + '&party=' + party + '&idCode=' + idCode,
		type:'POST',
		data:{
			action: 'create'
		},
		success: function(){
			window.added(true);
			window.newChoices();
		}
	});
}

function cancelCandidate(idCode) {
	$.ajax({
		url:'/cancelCandidate?idCode=' + idCode,
		type:'POST',
		data:{
			action: 'delete'
		},
		success: function(){
//			alert('Valisid kandidaadi nr: ' + candidateId);
			window.added(false);
			window.newChoices();
		}
	});
}
